scripts/interactions.js

```diff
@@ -61,14 +61,25 @@ function keypress(e){
 	var unicode=e.keyCode? e.keyCode : e.charCode;
 	console.log(unicode);
 	if(unicode === 46){ // Delete
-		// REMOVE SELECTION (from local graph only, this is not a deletion in remote database)
-		//for(node in graph.selection)
-		for(var i=0; i< graph.selection.length;i++)
+		if(graph.selection.length > 0)
 		{
```

Comment:
instead of having big blocks after a condition, you can use return if you only need to exit if the condition is not fullfilled